src/app/routes/utils/constructPageFetchUrl/index.ts

```diff
@@ -214,9 +231,13 @@ const constructPageFetchUrl = ({
         const variantPath = variant ? `/${variant}` : '';
         const host = `http://${process.env.HOSTNAME || 'localhost'}`;
         const port = process.env.PORT ? `:${process.env.PORT}` : '';
-        fetchUrl = Url(
-          `${host}${port}/api/local/${service}/live/${id}${variantPath}`,
-        );
+        let constructedUrl = `${host}${port}/api/local/${service}/live/${id}${variantPath}`;
+        const serviceLivePath = `${service}/live/`;
+        if (id.includes(`${service}`) && id.includes('live')) {
+          constructedUrl = constructedUrl.replace(serviceLivePath, '');
+        }
+        constructedUrl = constructedUrl.replace(/\/{2,}/g, '/');
```

Comment:
@LilyL0u , @louisearchibald and I have had a discussion offline about how this could be simplified, using the `path` instead of `id` + regexes